### ISSUE TITLE
[DATABASE]  Data base Schema

### ISSUE THREAD

**Issue by uo288524** [Arquisoft/wiq_es1c#100]
Define the data base schema of the application.

**Comment by gs-Manuel** [comment]
## State of Database
We are facing a problem regarding the database.

When implementing a microservices architecture, we are unclear how to define the database schema. We are considering two options:

- Single schema for the entire application: All services would communicate with a single database, which would act as another service, similar to the Blackboard pattern.
-  Pure microservices model: Each microservice would have its own independent database.

Due to this issue, the database schema is currently on hold. We need to define the architecture we will use for data persistence before we can implement the necessary schema(s).

In any case, we have decided to use a common [Miro diagram](https://miro.com/app/board/uXjVN9ntiDg=/?share_link_id=843234349083) to define the data model, which we will need regardless of the architecture chosen.
